types/seeed/senseCAPS2102/uplink.js

```diff
@@ -1,11 +1,25 @@
+function checkForIllegalValue(value) {
+  if (
+    value === 2000001 ||
+    value === 2000002 ||
+    value === 2000003 ||
+    value === 2000257
+  ) {
+    return null;
+  }
+  return value;
+}
+
 function consume(event) {
   const payload = event.data.payloadHex;
   const data = {};
+  const error = {};
   let pointer = 0;
 
   while (pointer < payload.length) {
     let measurementID = payload.substr((pointer += 2), 4);
     measurementID = Hex.hexLittleEndianToBigEndian(measurementID, false);
+    let value = 0;
```

Comment:
try to find a better way instead of reusing this variable (e.g. directly emit an error sample if it is an invalid value in checkForIllegalValue